### PR TITLE
ssr-apis.md

### PR DESCRIPTION
**Pull request by syshcndr**
Hey @spielers! I have updated "ssr-apis.md". Please review and take into consideration for Hacktoberfest label.

### CODE REVIEW

File: docs/docs/ssr-apis.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 title: Gatsby Server Rendering APIs
```
Comment (spielers) [inline docs/docs/ssr-apis.md]: Server Rendering = > translate this to Marathi!

File: docs/docs/ssr-apis.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 jsdoc: ["gatsby/cache-dir/api-ssr-docs.js"]
 apiCalls: SSRAPI
 showTopLevelSignatures: true
 ---
 
-## Usage
+## वापर
 
-Implement any of these APIs by exporting them from a file named `gatsby-ssr.js` in the root of your project.
+आपल्या प्रकल्पाच्या मुळाशी `gatsby-ssr.js` नावाच्या फाईलमधून निर्यात करुन यापैकी कोणतीही APIs अंमलात आणा.
```
Comment (spielers) [inline docs/docs/ssr-apis.md]: अंमलात आणा = >वापरात आणू  शकता

**Review by spielers — CHANGES_REQUESTED** [review]
Hey @syshcndr  Thanks for your contribution but I would suggest you check the required changes; once that done I will add the hacktoberfest accepted label to it; In case if you are aiming for hacktoberfest 

Best,
spielers

File: docs/docs/ssr-apis.md (hunk 2 above)
Comment (syshcndr) [reply docs/docs/ssr-apis.md]: Done.

**Review by spielers — APPROVED** [review]
Hey, @syshcndr Thanks for your contributions Will get you merged!
Hope you will like Gatsby swag!

Best,
spielers

**Pull request merged**